tests/FSharp.Data.Tests/JsonProvider.fs

```diff
@@ -480,13 +480,13 @@ let ``Can parse UTC dates``() =
     dates.UtcTime.ToUniversalTime() |> should equal (new DateTime(1997, 7, 16, 19, 50, 30, 0)) 
 
 [<Test>]
-[<SetCulture("zh-CN")>]
+[<Culture("zh-CN")>]
```

Comment:
This doesn't have the same effect. The previous attribute would switch to that culture when running the test and then switch back at the end, the new one just ignores the test unless you're running on that culture, which is not what we want. Output when running the tests on this branch:

      When running the tests I get this:
      Not Run  Can parse ISO 8601 dates in the correct culture
      Error Message:
       Only supported under culture zh-CN

      Not Run  Can parse ISO 8601 dates in the specified culture
      Error Message:
       Only supported under culture pt-PT

      Not Run  Can parse dates in different cultures
      Error Message:
       Only supported under culture pt-PT

      Not Run  Can parse decimals in different cultures
      Error Message:
       Only supported under culture pt-PT

      Not Run  Can parse document with iso date in local culture
      Error Message:
       Only supported under culture zh-CN

      Not Run  Can parse floats in different cultures
      Error Message:
       Only supported under culture pt-PT